docs/reference/commandline/ps.md

```diff
@@ -24,6 +24,7 @@ weight=1
       -q, --quiet=false     Only display numeric IDs
       -s, --size=false      Display total file sizes
       --since=""            Show created since Id or Name, include non-running
+      -F, --format=[]       Pretty-print containers using a Go template
```

Comment:
This was part of the PR I carried over, so I hadn't looked at it that closely and assumed that since review was approved it was reasonable.

However, looking at `inspect`, it does seem weird to have a short `-F` for **ps** but only the long option for **inspect**.  I'm definitely willing to drop `-F` from **ps** for uniformity, but I'm not sure how broad a review we want/need for that?  @docker/maintainers